### PR TITLE
Elakkiya/lang2

### PR DESCRIPTION
**Pull request by ElakkiyaMohanRajan**
What does this PR do? Please provide some context

User can able to select the Microsoft listed Preferred Language in the registration page and also he can modify the Preferred Language in Account Settings .

Where should the reviewer start?

Try to register as a new user , you have the ability to select the preferred language.

How can this be manually tested? (brief repro steps)

Before: No option to select the preferred language while registering and in account settings page
Now: Having the option to select the preferred language and if needed we can select the preferred language in Account Settings Page

What are the relevant TFS items? (list id numbers)
User Story 85071

#### Definition of done:
- [ ] Title of the pull request is clear and informative
- [ ] Add pull request hyperlink to relevant TFS items
- [ ] For large or complex change: schedule an in-person review session
- [ ] This change has appropriate test coverage

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging from a feature branch into the development branch then "flatten" or "squash" commits
1. If merging from the development branch into master (or porting changes from upstream) then use github's UI to get review feedback, but use the git command line interface to complete the actual merge.

#### Reminders AFTER merging
1. Delete the remote feature branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged from the development branch into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc). In this case, use github's UI for feedback and the git command line interface for the actual merge.

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 2 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 3 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 4 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

File: lms/envs/aws.py [hunk 1]
```diff
@@ -906,3 +907,16 @@
 _COUNTRIES_OVERRIDE = ENV_TOKENS.get('COUNTRIES_OVERRIDE', None)
 if _COUNTRIES_OVERRIDE:
     COUNTRIES_OVERRIDE = _COUNTRIES_OVERRIDE
+
+# all-languages overrides
+ALL_LANGUAGES_OVERRIDE = ENV_TOKENS.get('ALL_LANGUAGES_OVERRIDE', None)
+if ALL_LANGUAGES_OVERRIDE:
+    from collections import OrderedDict
```
Comment (sdolenc) [inline lms/envs/aws.py]: please bring these import statements to the top of the file (closer to the other imports). this is a style guideline i'm blocking on

File: lms/envs/aws.py [hunk 2]
```diff
@@ -906,3 +907,16 @@
...
+    from operator import itemgetter
+    for code, language in ALL_LANGUAGES_OVERRIDE.iteritems():
+         if language is None or language == "null":
+              del ALL_LANGUAGES_DICT[code]
+         elif code not in ALL_LANGUAGES_DICT:
+              ALL_LANGUAGES_DUPLICATE = ALL_LANGUAGES_DUPLICATE + ([code, language],)
```
Comment (sdolenc) [inline lms/envs/aws.py]: please add a comment in under the elif explaining a) the condition we're in and b) the work we're trying to do

File: lms/envs/aws.py [hunk 3]
```diff
@@ -906,3 +907,16 @@
...
+              ALL_LANGUAGES_DICT[code] = language
+    ALL_LANGUAGES_DICT = OrderedDict(sorted(ALL_LANGUAGES_DICT.items(), key=itemgetter(1)))
```
Comment (sdolenc) [inline lms/envs/aws.py]: i didn't know about OrderedDict. that's awesome 👍

File: lms/envs/aws.py (hunk 1 above)
Comment (sdolenc) [inline lms/envs/aws.py]: thanks!

File: lms/envs/aws.py (hunk 2 above)
Comment (ElakkiyaMohanRajan) [inline lms/envs/aws.py]: Added the comment and moved the import statement to the top. Please review it .

File: lms/envs/aws.py (hunk 3 above)
Comment (ElakkiyaMohanRajan) [inline lms/envs/aws.py]: Thank You

**Pull request merged**